lapce-app/src/window_tab.rs

```diff
@@ -1063,7 +1063,7 @@ impl WindowTabData {
                 self.palette.run(PaletteKind::Line);
             }
             Palette => {
-                self.palette.run(PaletteKind::File);
+                self.palette.run(PaletteKind::HelpAndFile);
```

Comment:
`Palette` should be `PaletteKind::File` I think. 

`PaletteKind::HelpAndFile` should only kicked in when you mouse click the search bar.

Reply:
Okay, I'll keep it on my private branch.